code/modules/operator/main.dm

```diff
@@ -0,0 +1,1024 @@
+
+	//support function vars
+	var/terminal = 0 //suit computer is disabled
+	var/o_stealth = 0 //suit cloaking disabled
+	var/o_busy = 0 //the suit isn't multithreaded like this code
+	var/overdrive = 0 //if overdrive is on
+	var/o_unlock = 0 //kamakazi on lockdown
+
+	var/s_bombs = 10 //number of smoke grenades
+	var/a_boost = 20 //number of adrenaline boosters
+	//legacy shit that exist so everything doesn't break
+	var/flush = 0 //DON'T YOU
+	var/o_control = 1 //WORRY ABOUT ME
+
+
+
+
+/obj/item/clothing/head/helmet/space/operator
+	name = "Polyweave Helmet and Balaclava"
+	desc = "A lightweight helmet and Balaclava combo that protects the user from space and other dangerous things. Issued and used by Cyber Connections operators"
+	armor = list(melee = 65, bullet = 30, laser = 60, energy = 15, bomb = 10, bio = 100, rad = 100)
+	body_parts_covered = HEAD
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+	w_class = 0
+
+/obj/item/clothing/mask/gas/voice/operator
+	name = "gas mask"
+	desc = "A close-fitting mask that connects to an air supply. Has a voice disguising feature. Issued and used by Cyber Connections operators."
+	icon_state = "s-ninja"
+	item_state = "s-ninja_mask"
+	vchange = 1
+	siemens_coefficient = 0.2
+
+/obj/item/clothing/gloves/operator
+	desc = "These nano-enhanced gloves insulate from electricity and provide fire resistance."
+	name = "cyber gloves"
+	icon_state = "s-ninja"
+	item_state = "s-ninja"
+	siemens_coefficient = 0.2
+	var/draining = 0
+	var/candrain = 0
+	var/mindrain = 200
+	var/maxdrain = 400
+
+/obj/item/clothing/shoes/operator
+	name = "cyber shoes"
+	desc = "A pair of advanced shoes that utilize <span_class='confirm'><B>NANOMACHINES, SON</B></span> to prevent the operator's environment from dicking them about."
+	icon_state = "s-ninja"
+	permeability_coefficient = 0.01
+	flags = NOSLIP
+	var/magpulse = 1 //Perma, non-slow magboots. Sorry ZAS!
+	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
+	siemens_coefficient = 0.2
+
+	cold_protection = FEET
+	min_cold_protection_temperature = SHOE_MIN_COLD_PROTECTION_TEMPERATURE
+	heat_protection = FEET
+	max_heat_protection_temperature = SHOE_MAX_HEAT_PROTECTION_TEMPERATURE
+/*---------------------------------------------------------------------------------------
+
+Clothing defines!
+
+---------------------------------------------------------------------------------------*/
+
+/*---------------------------------------------------------------------------------------
+
+//////////////////        VERBS!          ///////////////////////////////////////////////
+
+---------------------------------------------------------------------------------------*/
+
+/obj/item/clothing/suit/space/operator/New()//for when a new suit is made!
+	..()
+	verbs += /obj/item/clothing/suit/space/operator/proc/init //suit init verb
```

Comment:
giving a verb that calls a proc that does nothing but call another proc.

it's called ironic coding. GOD. *vaping intensifies*